yoti_python_sdk/activity_details.py

```diff
@@ -19,19 +21,23 @@ def __init__(self, receipt, decrypted_profile=None):
                     field.value,
                     field.content_type
                 )
-                self.user_profile[field.name] = value
 
-                if field.name == 'selfie':
+                anchors = Anchor().parse_anchors(field.anchors)
+
+                self.profile[field.name] = Attribute(field.name, value, anchors)
+                self.user_profile[field.name] = value  # will be removed in v3.0.0
+
+                if field.name == config.ATTRIBUTE_SELFIE:
                     self.try_parse_selfie_field(field)
 
                 if field.name.startswith(config.ATTRIBUTE_AGE_OVER) or field.name.startswith(
```

Comment:
As I said in the other PR:
I would rather use a regex here to capture age_(Over|Under) as there could be more derivation on age in the future.